### PR TITLE
Fix Two Tanks Showing for Input Hatches and Generators in TOP

### PR DESCRIPTION
**Pull request by BraggestSage833**
**What:**
Fix the issue where both fluid input hatches and single block generators would show two tanks in TOP instead of one.

**How solved:**
Used the initialize Inventory method to consolidate the input and output tank into one.

**Possible compatibility issue:**
None

**Additional info:**
I have also added a variable for setting tank size for all single block generators.

### CODE REVIEW

**Review by Yefancy — APPROVED** [review]

**Comment by ALongStringOfNumbers** [comment]
This fix also needs to be applied to MultiFluidHatches

File: src/main/java/gregtech/api/metatileentity/SimpleGeneratorMetaTileEntity.java [hunk 1]
```diff
@@ -63,6 +63,16 @@ protected FluidTankList createExportFluidHandler() {
         return new FluidTankList(false, this.importFluids);
     }
 
+    @Override
```
Comment (PrototypeTrousers) [inline src/main/java/gregtech/api/metatileentity/SimpleGeneratorMetaTileEntity.java:66]: Overriding the fluidInventory with the input tank looks enough to me
```    
protected void initializeInventory() {
        super.initializeInventory();
        this.fluidInventory = importFluids;
}
```

File: src/main/java/gregtech/common/metatileentities/electric/multiblockpart/MetaTileEntityFluidHatch.java [hunk 2]
```diff
@@ -58,6 +58,15 @@ public NBTTagCompound writeToNBT(NBTTagCompound data) {
         //fluidTank.writeToNBT(data);
         return data;
     }
+    @Override
```
Comment (PrototypeTrousers) [inline src/main/java/gregtech/common/metatileentities/electric/multiblockpart/MetaTileEntityFluidHatch.java:61]: same here.
Overriding the fluidInventory with the input tank looks enough to me
```    
protected void initializeInventory() {
        super.initializeInventory();
        this.fluidInventory = importFluids;
}
```

**Comment by ALongStringOfNumbers** [comment]
I forgot that this fix should also apply to the Steam Hatches

**Review by ALongStringOfNumbers — APPROVED** [review]

**Comment by Yefancy** [comment]
Need conflicts fixed

**Pull request closed without merging**